docs/semgrep-app/rule-board.md

```diff
@@ -41,6 +41,10 @@ Semgrep App detects the framework and language when scanning a project and only
 5. Drag the card and drop it on the relevant column.
 6. Once you are done adding rules and rulesets, click **Save changes**.
 
+:::tip
+Discover suggested rulesets based on your projects's frameworks and technologies through **Rule recommendations**. These recommendations appear when opening the drawer, enabling the user to choose from a curated list. Only rulesets explicitly added to a column will be included in a scan. To view these **Rule recommendations**, see [Running a scan](../getting-started-with-semgrep-app/#running-a-scan).
```

Comment:
“rule recommendations” isn’t capitalized above so for consistency the suggestion here is:

```suggestion
Discover suggested rulesets based on your projects's frameworks and technologies through **rule recommendations**. These recommendations appear when opening the drawer, enabling the user to choose from a curated list. Only rulesets explicitly added to a column will be included in a scan. To view these **rule recommendations**, see [Running a scan](../getting-started-with-semgrep-app/#running-a-scan).
```